server.js

```diff
@@ -20,11 +13,38 @@ app.use(cors());
 app.use(express.json());
 
 // Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+app.get('/', (req, res) => {
+	res.send('Golden Globe Winners');
+});
+
+//Fetches data from the API//
+app.get('/goldenGlobes', (req, res) => {
+	res.status(200).json(goldenGlobesData);
+});
+
+//Films by year//
+app.get('/goldenGlobes/year_award/:year_award', (req, res) => {
```

Comment:
When I use the route for year_award  and fill in for example
 /goldenGlobes/year_award/2010 
 I only get success: true but it doesn't show any data...